src/components/data-table/utils/sorting.tsx

```diff
@@ -0,0 +1,43 @@
+import { TableObject, Order, TablePagination, TableSorting } from "../types";
+
+function desc<T>(a: T, b: T, orderBy: keyof T) {
+    if (b[orderBy] < a[orderBy]) {
+        return -1;
+    }
+    if (b[orderBy] > a[orderBy]) {
+        return 1;
+    }
+    return 0;
+}
+
+function sort(array: TableObject[], cmp: (a: TableObject, b: TableObject) => number) {
```

Comment:
I do not particularly like this sorting technique, I used the default sorting of the original material-ui example but we can improve it.

Reply:
Yes, this a/b sorting is a bit clunky, better `_.sortBy`, or for a more sql aproach, `_.orderBy`.